greenlight/lib/api/prefs.py

```diff
@@ -2,109 +2,174 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from marionette.errors import MarionetteException
+
 from ..base import BaseLib
 
 
-class DefaultPrefBranch(BaseLib):
+class Preferences(BaseLib):
     archive = {}
 
-    @classmethod
-    def _cast(cls, value):
-        """
-        Interpolate a preference from a string.
+    def clear_user_pref(self, pref_name):
+        """Clear a user set preference.
 
-        - integers will get cast to integers
-        - true/false will get cast to True/False
-        - anything enclosed in single quotes will be treated as a string with
-          the ''s removed from both sides
+        :param pref_name: The preference to clear
         """
+        with self.marionette.using_context('chrome'):
+            return self.marionette.execute_script("""
+              Cu.import("resource://gre/modules/Services.jsm");
+              let prefBranch = Services.prefs.QueryInterface(Ci.nsIPrefBranch);
```

Comment:
Good catch! You are right, it also QI's it itself. So we should be able to remove that.